SYCL/DeviceLib/cmath_test.cpp

```diff
@@ -1,5 +1,3 @@
-// FIXME unsupported on windows (opencl) until fix of libdevice fails
-// UNSUPPORTED: windows && opencl
```

Comment:
This seems unrelated

Reply:
it was added for testing new action (this change exposed crash).